audio/softsynth/opl/nuked.h

```diff
@@ -30,26 +30,24 @@
 #define AUDIO_SOFTSYNTH_OPL_NUKED_H
 
 #include "common/scummsys.h"
+#include "common/inttypes.h"
 #include "audio/fmopl.h"
 
 #ifndef DISABLE_NUKED_OPL
 
+#ifndef OPL_ENABLE_STEREOEXT
+#define OPL_ENABLE_STEREOEXT 0
+#endif
+
 #define OPL_WRITEBUF_SIZE   1024
 #define OPL_WRITEBUF_DELAY  2
 
 namespace OPL {
 namespace NUKED {
 
-typedef uint            Bitu;
-typedef int             Bits;
-typedef uint64          Bit64u;
-typedef int64           Bit64s;
-typedef uint32          Bit32u;
-typedef int32           Bit32s;
-typedef uint16          Bit16u;
-typedef int16           Bit16s;
-typedef uint8           Bit8u;
-typedef int8            Bit8s;
+typedef uint64 uint64_t;
+typedef uint32 uint_fast32_t;
+#define UINT64_C(value) value##ui64
```

Comment:
These should be defined in `common/inttypes.h` with the rest of the fallbacks rather than here.